packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -2866,12 +2956,42 @@ class RepaintBoundary extends SingleChildRenderObjectWidget {
   RenderRepaintBoundary createRenderObject(BuildContext context) => new RenderRepaintBoundary();
 }
 
+/// Is invisible during hit testing.
```

Comment:
Yeah, this came from a misguided attempt to make the render object and widget text the same.